src/site/content/en/blog/codelab-stories-component/index.md

```diff
@@ -242,16 +376,16 @@ const state = {
 }
 ```
 
+<!-- TODO(kayce): All JS code needs semicolons: https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required -->
```

Comment:
:shudders: lol, jk, sgtm

Reply:
oh yeah, my style is "what don't i have to write".. there's going to be **more than semicolons** needing added to my js  😂

Reply:
That's the only big contentious one that I can see because of the risk of introducing bugs if the implicit semicolon insertion doesn't work as expected